src/org/opensolaris/opengrok/index/Indexer.java

```diff
@@ -757,7 +766,11 @@ public void prepareIndexer(RuntimeEnvironment env,
                     // This is an existing object. Reuse the old project,
                     // possibly with customizations, instead of creating a
                     // new with default values.
-                    projects.put(name, oldProjects.get(name));
+                    Project p = oldProjects.get(name);
+                    // Set the path which may be missing in the project if
+                    // the project was customized.
+                    p.setPath(path);
```

Comment:
what about `tabSize` ? Perhaps more generic approach with filling in all fields (copy constructor ?) would be in order.